index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
           <!-- Keep one line of space above and below your card -->
           <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+<!-- ________ Revathi card START ________  -->
+          <div class="card">
+              <p class="name">Revathi</p>
+              <p class="contact">
+                  <i class="fab fa-twitter"></i>
+                  <a href="https://www.twitter.com/" target="_blank">@Revathi_tw</a>
```

Comment:
```suggestion
                  <a href="https://www.twitter.com/Revathi_tw" target="_blank">@Revathi_tw</a>
```
your social link should contain your profile URL